README.md

```diff
@@ -146,3 +146,28 @@ dotnet build -c Release
 **Unity3D**
 
 If you wish to have barebones Unity3D implementation, you need to build the `DiscordRPC.dll`, the [Unity Named Pipes](https://github.com/Lachee/unity-named-pipes) Library and the [UnityNamedPipe.cs](https://github.com/Lachee/discord-rpc-csharp/blob/master/Unity%20Example/Assets/Discord%20RPC/Scripts/Control/UnityNamedPipe.cs). Put these in your own Unity Project and the `.dll`s in a folder called `Plugins`. 
+
+**UWP / .NET MAUI / WIN UI 3**
+
+In order to make this library work with the UWP applications, you need to define `runFullTrust` Capability inside of `Package.appxmanifest`. It's done this way because library loads external .exe in order to make library work.
```

Comment:
this library does not load exeternal exes, or any exes actually. Please reword this.

some potential causes could be:
- use of registry keys in [WindowsUriSchemeCreator.cs](https://github.com/Lachee/discord-rpc-csharp/blob/master/DiscordRPC/Registry/WindowsUriSchemeCreator.cs#L57-L72)
- use of pipes in [ManagedNamedPipeClient.cs](https://github.com/Lachee/discord-rpc-csharp/blob/master/DiscordRPC/IO/ManagedNamedPipeClient.cs#L142)

but unless verified what is causing it, please only mention that runFullTrust is required.

From reading the capability declarations, it could also be from .JSON using JIT capabilities, although this is not a restricted capability (`Code Generation`).